tools/mo/openvino/tools/mo/ops/lstm_sequence.py

```diff
@@ -60,6 +60,10 @@ def supported_attrs(self):
     def backend_attrs(self):
         return [
             'hidden_size',
+            ('activations', lambda node: ','.join(node['activations'])),
+            ('activations_alpha', lambda node: ','.join(map(str, node['activations_alpha']))),
+            ('activations_beta', lambda node: ','.join(map(str, node['activations_beta']))),
+            'direction',
         ]
```

Comment:
Should the attribute `clip` be added here?

Reply:
Yes it should, added.